scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py

```diff
@@ -320,13 +322,16 @@ def allow_any_int(i): return int(i, 0)
     optional_arguments.add_argument("--pai_cert", type=str,
                                     help="[.der] Provide the path to .der file containing PAI certificate.")
     optional_arguments.add_argument("--rd_uid", type=str,
-                                    help="[hex string] Provide the rotating device unique ID. If this argument is not provided a new rotating device id unique id will be generated.")
+                                    help="[hex string] [pattern: 'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'] Provide the rotating device unique ID. If this argument is not provided a new rotating device id unique id will be generated.")
```

Comment:
Not sure if this pattern thing is understandable. IMO, 128-bit **hex-encoded** value is good enough.